hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java

```diff
@@ -127,6 +129,7 @@ public static class Builder {
     private boolean isRatisEnabled = false;
     private boolean isTracingEnabled = false;
     private Function<Long, Long> indexToTerm = null;
+    private Supplier<List<Long>> nextIndexes = null;
```

Comment:
Please don't add `nextIndexes` supplier.  It will make the logic more complicated.  As mentioned, we should simplify the logic and remove the maps.